draft-packages/table/KaizenDraft/Table/Table.tsx

```diff
@@ -20,23 +21,32 @@ type TableContainerProps = {
  */
 export const TableContainer = ({
   variant = "compact",
+  caption,
   children,
   ...otherProps
 }: TableContainerProps): JSX.Element => (
-  <div
-    role="table"
+  <table
     className={classNames(styles.container, {
       [styles.defaultSpacing]: variant === "default",
       [styles.dataVariant]: variant === "data",
     })}
     {...otherProps}
   >
+    {caption && (
+      <Heading
+        classNameOverride="pb-12 text-left"
```

Comment:
Unfortunately you cannot use Tailwind here, as there are code bases that don't have Tailwind configured, and this lives outside the `components` package that enables Tailwind distribution.

Regardless, this should be a margin.

Reply:
Yep sure, I'll do that. I did try `mb-12` but due to a specificity issue it gets overridden by the Heading's `margin: 0;`. But not using Tailwind should resolve that.

Reply:
@gyfchong I've added the changes you requested in the latest commit.